lib/expander/src/Events/CourseModuleCompleted.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: nikolay.mikov
+ * Date: 5.6.2017 г.
+ * Time: 11:32
+ */
+
+namespace LogExpander\Events;
+
+
+class CourseModuleCompleted extends  Event
+{
+    public function read(array $opts)
+    {
+
+        global $DB;
```

Comment:
Like @garemoko said you can just make new commits to this branch at https://github.com/nikidev/moodle-logstore_xapi/tree/course_module_completed_event_finished